src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/ReclusterServiceIntTest.kt

```diff
@@ -2,502 +2,212 @@ package uk.gov.justice.digital.hmpps.personrecord.service
 
```

Comment:
As reclustering now happens solely as a result of update processing, I think it would be sensible to have at least one test which proves the whole process - consume an update message and verify that reclustering happens as a result.